source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -167,6 +167,16 @@ nodes in the deployment at around the same time.
    :start-after: start-nondisruptive-upgrade-desc
    :end-before: end-nondisruptive-upgrade-desc
 
+Behavior
+--------
+
+Final Listing Check
+~~~~~~~~~~~~~~~~~~~
+
+At the end of the decommission process, MinIO checks for a list of items on the pool.
+If any objects return, MinIO returns an error that the decommission process failed.
```

Comment:
QQ: Does the user just repeat decom at this point? Would be ideal to give user guidance here.

Reply:
Got some feedback from Harsha that I'll apply, then merge.